app/controllers/EightBallController.scala

```diff
@@ -0,0 +1,30 @@
+package controllers
+
+import javax.inject._
+
+import play.api.mvc._
+
+import models.EightBallReply
+
+@Singleton
+class EightBallController @Inject()(cc: ControllerComponents) extends AbstractController(cc) {
+
+  def randomReply = Action {
+    val rndString = EightBallReply.getRndBall.content
+    Ok(views.txt.text(rndString))
```

Comment:
You can replace the parameter of all of these calls to `Ok` with passing the string directly.